distribution/pom.xml

```diff
@@ -252,9 +252,9 @@
                                 <zip destfile="target/wso2carbon-core-${carbon.kernel.version}/wso2/components/plugins/org.wso2.carbon.security.mgt.ui_${carbon.identity.framework.version}.jar" basedir="${tempdir2}" />
 
                                 <!--Move solr registry -->
-                                <unzip src="target/wso2carbon-core-${carbon.kernel.version}/wso2/components/plugins/org.wso2.carbon.registry.indexing_${carbon.registry.version}.jar" dest="${tempdir2}" />
+                                <unzip src="target/wso2carbon-core-${carbon.kernel.version}/wso2/components/plugins/org.wso2.carbon.registry.indexing_4.6.42.SNAPSHOT.jar" dest="${tempdir2}" />
                                 <replace file="${tempdir2}/core.properties" token="dataDir=../../../../solr/data" value="dataDir=../../../solr/data" />
-                                <zip destfile="target/wso2carbon-core-${carbon.kernel.version}/wso2/components/plugins/org.wso2.carbon.registry.indexing_${carbon.registry.version}.jar" basedir="${tempdir2}" />
+                                <zip destfile="target/wso2carbon-core-${carbon.kernel.version}/wso2/components/plugins/org.wso2.carbon.registry.indexing_4.6.42.SNAPSHOT.jar" basedir="${tempdir2}" />
```

Comment:
you shouldn't put snapshot here

Reply:
Once we release the kernel we can remove the snapshot.